kube/foo/generic-job.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: holdings-client-$SUFFIX
+  namespace: holdings
+spec:
+  template:
+    spec:
+      containers:
+      - command: $ARGS
+        env:
+        - name: MONGODB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: mongodb-password
+              name: holdings-mongodb
+        - name: MYSQL_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: mysql-password
+              name: holdings-mysql
+        - name: MONGOID_ENV
+          value: production
+        - name: TZ
+          value: America/Detroit
+        image: docker.pkg.github.com/hathitrust/holdings/client-unstable
+        imagePullPolicy: Always
+        name: holdings-client
+        volumeMounts:
+        - mountPath: /htapps
+          name: htapps
+        - mountPath: /htapps-dev
+          name: htapps-dev
+        - mountPath: /usr/src/app/config/settings/production.local.yml
+          name: holdings-config
+          subPath: production.local.yml
+        - mountPath: /usr/src/app/config/large_cluster_ocns.txt
+          name: large-cluster-ocns
+          subPath: large_cluster_ocns.txt
+        - mountPath: /htprep
+          name: htprep
+      restartPolicy: OnFailure
+      securityContext:
+        fsGroup: 1190
+        runAsGroup: 1191
+        runAsUser: 1000
```

Comment:
Are these magic and confirmed good? Or arbitrary?

Reply:
Magic and confirmed good (matches groups assigned on the NFS mounts)